pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -121,3 +135,17 @@ func removeEmptyStrings(s []string) []string {
 	}
 	return nonempty
 }
+
+//GenerateSecureString creates a webhook secret for you.
```

Comment:
Is that really what `GenerateSecureString` is doing?

Also, should it be exported?  I tend to fall on the side of not exporting things until I find a need to export.

Nitpick: I think this should have a space between `//` and `Generate`

Reply:
Comment modified to match the purpose of the function.